/**:
  ros__parameters:
    simulation_dt: 0.002
    max_torque: 0.43
    max_velocity_joint0: 22.0
    max_velocity_joint1: 100.0

    initial_conditions:
      theta1: 0.0
      theta2: 0.0

      dtheta1: 0.0
      dtheta2: 0.0

      tau1: 0.0
      tau2: 0.0
